src/overlays/actors/ovl_En_Dodojr/z_en_dodojr.c

```diff
@@ -405,59 +405,59 @@ void func_809F73AC(EnDodojr* this, PlayState* play) {
             this->actor.shape.rot.x = this->actor.world.rot.x;
             this->dustPos = this->actor.world.pos;
             this->dustPos.y = this->actor.floorHeight;
-            this->actionFunc = func_809F74C4;
+            this->actionFunc = EnDodojr_EmergeFromGround;
         }
     }
 }
 
-void func_809F74C4(EnDodojr* this, PlayState* play) {
+void EnDodojr_EmergeFromGround(EnDodojr* this, PlayState* play) {
     f32 sp2C;
 
     Math_SmoothStepToS(&this->actor.shape.rot.x, 0, 4, 0x3E8, 0x64);
     sp2C = this->actor.shape.rot.x;
     sp2C /= 16384.0f;
     this->actor.world.pos.y = this->actor.home.pos.y + (60.0f * sp2C);
-    func_809F6510(this, play, 3);
+    EnDodojr_SpawnLargeDust(this, play, 3);
 
     if (sp2C == 0.0f) {
         this->actor.shape.shadowDraw = ActorShadow_DrawCircle;
         this->actor.world.rot.x = this->actor.shape.rot.x;
         this->actor.speedXZ = 2.6f;
-        this->actionFunc = func_809F758C;
+        this->actionFunc = EnDodojr_CrawlTowardsTarget;
     }
 }
 
-void func_809F758C(EnDodojr* this, PlayState* play) {
+void EnDodojr_CrawlTowardsTarget(EnDodojr* this, PlayState* play) {
     func_8002D868(&this->actor);
-    func_809F6730(this, play, &this->actor.world.pos);
+    EnDodojr_SpawnSmallDust(this, play, &this->actor.world.pos);
 
-    if (DECR(this->timer4) == 0) {
+    if (DECR(this->crawlSfxTimer) == 0) {
         Audio_PlayActorSound2(&this->actor, NA_SE_EN_DODO_M_MOVE);
-        this->timer4 = 5;
+        this->crawlSfxTimer = 5;
     }
 
-    if (func_809F6DD0(this) != 0) {
-        func_809F6C24(this);
-        this->actionFunc = func_809F768C;
+    if (EnDodojr_CanEatBomb(this) != 0) {
```

Comment:
```suggestion
    if (EnDodojr_CanEatBomb(this)) {
```

Reply:
I keep forgetting that changes like this will still match!

Reply:
[4707dc3](https://github.com/zeldaret/oot/pull/1311/commits/4707dc32e76d0915f4a21d391fff5f419dcbb932)